soletta/builder.go

```diff
@@ -0,0 +1,42 @@
+package soletta
+
+/*
+#include <soletta.h>
+#include <sol-flow-builder.h>
+*/
+import "C"
+import "unsafe"
+
+type FlowBuilder struct {
+	builder *C.struct_sol_flow_builder
+}
+
+func (fb *FlowBuilder) Init() {
+	fb.builder = C.sol_flow_builder_new()
+}
+
+func (fb *FlowBuilder) AddNode(name string, nodeType string) {
+	cname, cnodeType := C.CString(name), C.CString(nodeType)
+	defer C.free(unsafe.Pointer(cname))
+	defer C.free(unsafe.Pointer(cnodeType))
+	C.sol_flow_builder_add_node_by_type(fb.builder, cname, cnodeType, nil)
+}
+
+func (fb *FlowBuilder) Connect(name1, port1, name2, port2 string) {
```

Comment:
in the previous function you did:
`name string, nodeType string`
here we are using the pattern:
`name, nodeType string`

Shouldn't you follow the same pattern in both ?